ios/FluentUI/PersonaButton/PersonaButtonTokens.swift

```diff
@@ -13,9 +13,9 @@ import UIKit
     var avatarSize: MSFAvatarSize {
         switch self {
         case .large:
-            return .xxlarge
+            return .size72
         case .small:
-            return .xlarge
+            return .size56
```

Comment:
These `Avatar` instances still appear as 52pt in Figma. Has design confirmed that this component should be changing as well?

Reply:
Right, the figmas don't seem to have been updated. I assume since design requested we change size 52 they should be aware any other control using 52pt avatar will change to 56pt avatar. I can check with design on Monday (since they're both oof) but this will probably not make it this fork then.

Reply:
Confirmed the changes with design.